icons/mask.svg

```diff
@@ -0,0 +1,16 @@
+<svg 
+    xmlns="http://www.w3.org/2000/svg"  
+    width="24" 
+    height="24" 
+    viewBox="0 0 24 24" 
+    fill="none" 
+    stroke="currentColor"
+    stroke-width="2"
+    stroke-linecap="round"
+    stroke-linejoin="round"
+>
+    <path d="M13.4756 7.29967L18 9.21398L17.8379 11.3549C17.6424 13.9345 15.8712 16.1109 13.4131 16.7917L13.2979 16.8235C12.4481 17.0588 11.5519 17.0588 10.7022 16.8235L10.587 16.7917C8.12886 16.1109 6.35759 13.9345 6.16218 11.3549L6 9.214L10.5244 7.29967C11.4688 6.90011 12.5312 6.90011 13.4756 7.29967Z" />
+    <path d="M18 9C18.5 7.99999 19 7 20 7C20.8798 7 22 7.5 22 9C22 10.5 21.7 11 20.5 13C19.3 15 17.4424 15.8219 15.5 16" />
+    <path d="M6 9C5.5 7.99999 5 7 4 7C3.12025 7 2 7.5 2 9C2 10.5 2.3 11 3.5 13C4.7 15 6.55762 15.8219 8.5 16" />
+    <path d="M9.5 11C11.0388 9.66667 12.9612 9.66667 14.5 11" />
```

Comment:
The nose wire still violates the 2px gap rule, removing it would work tho.

```suggestion
  <path d="m18 9.214-.162 2.14c-.196 2.58-1.967 4.757-4.425 5.438l-.115.031a4.86 4.86 0 0 1-2.596 0l-.115-.031c-2.458-.681-4.23-2.858-4.425-5.437L6 9.214 10.524 7.3a3.783 3.783 0 0 1 2.952 0z" />
  <path d="M18 9c.5-1 1-2 2-2 .88 0 2 .5 2 2s-.3 2-1.5 4-3.058 2.822-5 3" />
  <path d="M6 9c-.5-1-1-2-2-2-.88 0-2 .5-2 2s.3 2 1.5 4 3.058 2.822 5 3" />
```

Reply:
yes removed it